multicast/discovery_test.go

```diff
@@ -34,12 +43,48 @@ func (m *multicastSuite) Test_Lookup() {
 		{
 			desc:          "System with maximum allowed server name length, IPv6 address and high version number can be looked up",
 			lookupVersion: "142.0",
+			lookupIface:   "lo",
+			lookupPort:    9444,
 			responseInfo: ServerInfo{
 				Version: "142.0",
 				Name:    strings.Repeat("a", 255),
 				Address: "fd42:c4cc:2e1d:132d:a216:3eff:fecd:9d15",
 			},
 		},
+		{
+			desc:        "Cannot lookup system if invalid interface is given",
+			lookupIface: "invalid-interface",
+			lookupErr:   fmt.Errorf(`Failed to resolve lookup interface "invalid-interface": route ip+net: no such network interface`),
+		},
+		{
+			desc:          "Cannot lookup system if the responder is offline",
+			lookupVersion: "2.0",
+			lookupIface:   "lo",
+			lookupPort:    9444,
+			responseInfo: ServerInfo{
+				Version: "2.0",
+				Name:    "foo",
+				Address: "1.2.3.4",
+			},
+			lookupTimeout: 500 * time.Microsecond,
```

Comment:
Timeouts of 500 microseconds seem extremely short... I'm a bit worried about ir leading to flaky tests due to variability in system scheduling, garbage collection pauses (or other factors in the CI workers) . What do you think? Is `lookupTimeout: 500 * time.Millisecond` safer for example?

Reply:
Ah yeah actually my intention was to use `Millisecond` but it shouldn't really matter. As there is no server running (see the line below, it's getting stopped beforehand), we could also have a timeout of 0. So it's not like a sleep or something. 

In general I have tried to use a very short amount of time to not much influence the runtime of the test.